modules/ROOT/pages/configure-teams.adoc

```diff
@@ -0,0 +1,135 @@
+If you have the Organization Administrator permission, you can configure permissions for this team in addition to child teams.
+If you invite users to “Everyone at <Your Organization>” without assigning any permissions to that team, your users cannot execute tasks or view assets in your organization.
+
+If you have the Organization Administrator permission in a specific business group but not your entire organization, you can assign these permissions to any of existing teams or users within your organization.
+
+To configure permissions for teams in the Teams feature interface:
+
+include::partial$include-nav-steps-team-maintainer.adoc[]
+
+. Click the team for which you want to configure permissions.
+. Click the *Permissions* tab.
+. In the *Permissions* section, click *Add permissions*
+. In the *Select Permissions* window, select the permissions that you want to add, and then select *Select contexts*.
+. Select the business groups and environments to which you want to apply the permissions, and then select *Review*. +
+The permissions you selected appear alongside the business groups and environments to which they apply.
++
+. Select *Apply permissions*. +
+The permissions you selected appear in the *Permissions* section for your team.
+
+== Manage Team Users
+
+If you have the Organization Administrator permission in your root organization, you can add members to existing teams using three methods:
+
+* The Teams feature interface
+* Sending an invitation to a new user via email
+* External group mapping
+
+[[invite-users-with-teams]]
+=== Add Users using the Teams Interface
+
+include::partial$include-nav-steps-team-maintainer.adoc[]
+
+. In the *Teams* page, select the team to which you want to add a user.
+. Select *Add members*.
+. Select the name of each user you want to add to the team.
+. Select the membership status that you want to assign to each user: team member or team maintainer. + 
+A user who is added as a team maintainer receives team maintainer permissions as soon as you confirm their membership. 
++
+. Select *Add member*. +
+The user is added to the team and receives the team’s permissions.
+
+=== Add Users using Email Invitations
+
+When you send an email to invite a new user to your organization, you can specify teams to which the user is added automatically when they accept your invitation. The user is also automatically added to your organization’s root team, known as “Everyone at <Your Organization>” by default.
+
+For more information about inviting a user to an existing team via email, see xref:users.adoc#inviting-a-user-to-anypoint-platform[Inviting a User to Anypoint Platform].
+
+=== Add Users using External Group Mapping
+
+If you use single sign-on (SSO) using SAML, OIDC, or LDAP, you can map SSO users to teams. For more information, see xref:map-users-roles-teams.adoc[Map SSO Users to Teams].
+
```

Comment:
OK, I don't want to stress you out with all these edits, so I am going to ONLY CE from here on. Imma' assume these wobbly terms are made clear earlier? If not, I would like for you to create a JIRA and make an edit request sometime after the release, so I can look at the whole thing when we both have more time.